resolver-dns/src/test/java/io/netty/resolver/dns/DnsNameResolverTest.java

```diff
@@ -2759,6 +2760,25 @@ public void testTruncatedWithTcpFallbackBecauseOfMtu() throws IOException {
         testTruncated0(true, true);
     }
 
```

Comment:
@njhill can we also verify that this line returns false (add a `assertFalse`):
https://github.com/netty/netty/blob/a95d04bb2f6b5f9d000f22b4daa36b5b1f12935b/resolver-dns/src/test/java/io/netty/resolver/dns/DnsNameResolverTest.java#L2901

Reply:
@normanmaurer do you mean that it should return true (no more refs left)? I just tried with false and it fails :)

Reply:
yes sorry... `true` :)